manifests/l/lucasdeeiroz/RobotRunner/2.0.1/lucasdeeiroz.RobotRunner.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: lucasdeeiroz.RobotRunner
+PackageVersion: 2.0.1
+PackageLocale: en-US
+Publisher: LDE Software

```

```suggestion
Publisher: lucas
```
This should match what is written to control panel, or AppsAndFeaturesEntries should be specified in the installer manifest